hydradx/notebooks/Omnipool/TestPriceMoveWithdraw.ipynb

```diff
@@ -0,0 +1,585 @@
+      "text/plain": [
+       "<Figure size 1440x360 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "price_moves = {0.5: '-50%', 10: '10x'}\n",
```

Comment:
This was certainly a good question, because including this sale in the simulation reverses the price dynamic. I found that it doesn't matter much when the agent sells the LRNA, or what they buy with it. In any case, we go from a 16% price increase to about a 3% drop.